estudios_socioeconomicos/forms.py

```diff
@@ -0,0 +1,56 @@
+from django import forms
+
+from .models import Respuesta, OpcionRespuesta
+
+
+class RespuestaForm(forms.ModelForm):
+    """ Model form for answers
+
+        This is the general form for saving an answer.
+    """
+
+    def __init__(self, *args, **kwargs):
+        """ Overide __init__ method to add query to elections multiple select.
+
+            The 'elecciones' field in this form need a query set that depends
+            upon the specific question insance. (Certain question have)
+            predefined answer choices. This custom init recieves the instance
+            of the question this RespuestaForm will be answering to query
+            the options for that question.
+        """
+        pregunta = kwargs.pop('pregunta', None)
+        super(RespuestaForm, self).__init__(*args, **kwargs)
+        if pregunta:
+            self.fields['elecciones'].queryset = OpcionRespuesta.objects.filter(pregunta=pregunta)
+
+        if 'instance' in kwargs:
+            self.fields['elecciones'].initial = self.instance.elecciones
+
+    respuesta = forms.CharField(
+        required=False,
```

Comment:
This is actually a very good question, by design we could leave it like this, if we chose to make some mandatory we would have to add a field into the model. We should further discuss this with the team.